service/controller/key/cluster.go

```diff
@@ -14,7 +14,7 @@ import (
 const (
 	// CloudConfigVersion defines the version of k8scloudconfig in use. It is used
 	// in the main stack output and S3 object paths.
-	CloudConfigVersion = "v_5_2_0"
+	CloudConfigVersion = "v_6_0_0"
```

Comment:
It might be cool to export this from `k8scloudconfig` in the future. One less thing to change.

Reply:
Agreed - let's do that in a follow up PR though.